src/js/providers/video-listener-mixin.js

```diff
@@ -177,7 +177,7 @@ const VideoListenerMixin = {
 
         this.trigger(
             MEDIA_ERROR,
-            new PlayerError(`Error loading media ${errorData[0]}`, code, this.video.error)
+            new PlayerError(MSG_CANT_PLAY_VIDEO, code, this.video.error)
```

Comment:
we are no longer using the strings in `errorData`, so we could get rid of them. 'Unknown network error' i think maps best with `badConnection`

Reply:
we might want to rewrite this. Basically we map the HTML5 `video.error.code` to our own codes:

0->  224000
1->  224001
2-> 221000 (notice this is a `1000` level to indicate it is network specific, vs `4000` indicating it is media specific)
3 -> 224002
4 -> 224003
(> 4) -> 224000 (fallback to base code because the video element's error code is unknown)

Reply:
@karimJWP so 'File could not be played' is the only one that should be mapped to `MSG_CANT_PLAY_VIDEO ` now?